Doc/library/threading.rst

```diff
@@ -45,7 +91,23 @@ level :mod:`_thread` module.
    However, threading is still an appropriate model if you want to run
    multiple I/O-bound tasks simultaneously.
 
-.. include:: ../includes/wasm-notavail.rst
+GIL and Performance Considerations
```

Comment:
```suggestion
GIL and performance considerations
```
And we have a mix of headers on this page, I suggest updating the others to use sentence case as well.

https://devguide.python.org/documentation/style-guide/#capitalization